src/views/map/index.js

```diff
@@ -90,6 +92,17 @@ class Map extends Component {
             options={ () => this.mapOptions }
             onGoogleApiLoaded={ this.handleGoogleMapLoaded }
             yesIWantToUseGoogleMapApiInternals={ true }>
+            { /* display pokémon spots from pokévision */ }
+            { pokevision.pokemonSpots.map((p, idx) =>
+              <Pokemon
+                key={ idx }
```

Comment:
I guess this should be 'p.id' instead of 'idx', otherwise it messes up when pokemon spots are removed from the array.